modeltranslation/fields.py

```diff
@@ -156,8 +158,53 @@ def __init__(self, translated_field, language, empty_value, *args, **kwargs):
         # (will show up e.g. in the admin).
         self.verbose_name = build_localized_verbose_name(translated_field.verbose_name, language)
 
+        # M2M support - <rewrite related_name> <patch intermediary model>
+        if isinstance(self.translated_field, fields.related.ManyToManyField) and hasattr(
+            self.remote_field, "through"
+        ):
+            import copy
```

Comment:
Please, move this import to the top level

Reply:
Yep